packages/website/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@nl-design-system-community/website",
+  "type": "module",
+  "version": "0.0.1",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/nl-design-system/theme-wizard.git",
+    "directory": "packages/website"
+  },
+  "scripts": {
+    "dev": "astro dev",
+    "build": "pnpm run clean && astro build",
+    "prebuild": "pnpm -C ../stencil build",
```

Comment:
`pnpm -C`: ik heb geen idee wat dat doet 🤔 Is er een langere, self-documenting CLI option beschikbaar?

Also: is dit echt nodig om hier dependencies te gaan builden? In principe wil je dat door `pnpm` laten doen. Als je vanuit de workspace `pnpm run build` doet dat gaat deze volgorde al goed.

Als je scripts vanuit meerdere packages wilt coordineren, dan stel ik voor dat je dat doet vanuit de `/package.json`, en `pnpm` de volgorde laat regelen. Je kunt daar dan zeggen: voer script X uit in packages Y en Z